packages/plugin-ext/src/main/browser/env-main.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { interfaces } from "inversify";
+import { EnvVariablesServer } from "@theia/core/lib/common/env-variables";
+import { RPCProtocol } from '../../api/rpc-protocol';
```

Comment:
Let's use ' or "  for all import

Reply:
Missed it, probably autoimport. Thnaks for spotting